sunpriya/__init__.py

```diff
@@ -0,0 +1,45 @@
+from flask import Flask,jsonify,request,make_response
+import constants as constants
+from exception import CSSException
+app=Flask(__name__)
```

Comment:
Not enough spacing given between lines (as per PEP 8 guidelines). If you are using PyCharm, violations should be underlined automatically.